# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1310848387 14400
#      Sat Jul 16 16:33:07 2011 -0400
# Node ID ab6fc7b40541759cba6eb38b52aaf4b9209519bb
# Parent  46e0d66596f2c31e2af9f309ae5e75bc4a12ca3c
codesprint: add assert(1) test for __dispatch__

* __dispatch__.cc: No tests needed for internal function that
is only present to support deprecated function.

diff --git a/src/DLD-FUNCTIONS/__dispatch__.cc b/src/DLD-FUNCTIONS/__dispatch__.cc
--- a/src/DLD-FUNCTIONS/__dispatch__.cc
+++ b/src/DLD-FUNCTIONS/__dispatch__.cc
@@ -124,8 +124,15 @@ DEFUN_DLD (__dispatch__, args, nargout,
       else
         symbol_table::add_dispatch (f, t, r);
     }
   else
     print_usage ();
 
   return retval;
 }
+
+/*
+
+## No test needed for internal helper function.
+%!assert (1)
+
+*/
